cognite/powerops/resync/models/v2/dms/frontendContract/containers.yaml

```diff
@@ -0,0 +1,93 @@
+- space: {{model_space}}
+  externalId: Alert
+  name: Alert
+  usedFor: node
+  properties:
+    time:
+      type:
+        list: false
+        type: timestamp
+      nullable: false
+      autoIncrement: false
+      name: time
+      defaultValue: null
+      description: null
+    title:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: false
+      autoIncrement: false
+      name: title
+      defaultValue: null
+      description: null
+    description:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: description
+      defaultValue: null
+      description: null
+    severity:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: severity
+      defaultValue: null
+      description: null
+    alertType:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
```

Comment:
I think there is nothing technically wrong with indexing a nullable column. Not a best practice maybe, but since we need to decide what we want to make required anyways at some point, let's address it then. 
But that's definitely not a strong opinion. Did I miss something?